config/app.php

```diff
@@ -4,6 +4,31 @@
 
     /*
     |--------------------------------------------------------------------------
+    | Application Name
+    |--------------------------------------------------------------------------
+    |
+    | This value is the name of your application. This value is used when the
+    | framework needs to place the application's name in a notification or
+    | any other location as required by the application or its packages.
+    */
+
+    'name' => 'Mediviron',
```

Comment:
Another instance of business-specific variables. This should go in the .env file.